src/libsystemd/sd-bus/bus-introspect.c

```diff
@@ -101,6 +101,12 @@ static int introspect_write_arguments(struct introspect *i, const char *signatur
 
                 fprintf(i->f, "   <arg type=\"%.*s\"", (int) l, signature);
 
+                if(*names) {
+                    int ret = fprintf(i->f, " name=\"%s\"", names)-8;
```

Comment:
please follow CODING_STYLE. indentation is off, no space between `if` and `(`. please don't mix variable declarations and function invocations.